modules/update-restricted.adoc

```diff
@@ -34,13 +34,23 @@ The release image signature config map allows the Cluster Version Operator (CVO)
 +
 [source,terminal]
 ----
-$ oc adm upgrade --allow-explicit-upgrade --to-image ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}@<digest> <1>
+$ oc adm upgrade --allow-explicit-upgrade --to-image <defined_registry>/<defined_repository>@<digest>
 ----
-<1> The `<digest>` value is the sha256 digest for the targeted release image, for example, `sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`
 +
-If you use an `ImageContentSourcePolicy` for the mirror registry, you can use the canonical registry name instead of `LOCAL_REGISTRY`.
+Where:
++
+** `<defined_registry>` is the name of the mirror registry you mirrored your images to.
++
+** `<defined_repository>` is the name of the image repository you want to use on the mirror registry.
++
+** `<digest>` is the sha256 digest for the targeted release image, for example, `sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`.
 +
 [NOTE]
 ====
-You can only configure global pull secrets for clusters that have an `ImageContentSourcePolicy` object. You cannot add a pull secret to a project.
+* See "Mirroring {product-title} images" to review how your mirror registry and repository names are defined.
```

Comment:
`Mirroring {product-title} images` is one section of our document, shall we use a link here? or give more specified words, like `See "Mirroring {product-title} images" section in the same level ...`. Thanks. Thought I was new to OCP.

Reply:
I wish I could use a link right here, but there are constraints to our docs repo that prevent us from putting xref links in module files. So that's why I add an xref link in the assembly [right here].(https://github.com/openshift/openshift-docs/pull/68593/files#diff-b5fc9884acf533766901f793dc9e6a72794198b7e3bb0bca8a79f76447440a9fR51)